tools/hack/kind-expose-ports.yaml

```diff
@@ -0,0 +1,12 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
```

Comment:
Please rebase/merge with `main`; the yamllint config is going to complain about this whitespace.